modules/gapi/test/internal/gapi_int_pattern_matching_test.cpp

```diff
@@ -994,4 +998,1012 @@ TEST(PatternMatching, adeSmokeTest)
     EXPECT_EQ(2u, dst->inNodes().size());
 }
 
+// --------------------------------------------------------------------------------------
```

Comment:
It is bad idea to expand this file with new entities. Please move to another test file